Readme.md

```diff
@@ -4,19 +4,72 @@
 [![NPM Downloads][downloads-image]][downloads-url]
 [![Build Status][travis-image]][travis-url]
 
-Utility to parse a string bytes (ex: `1TB`) to bytes (`1099511627776`) and vice-versa.
+Utility to parse a string bytes (ex: `1TB`) to bytes (`1,000,000,000,000`) and vice-versa.
+
+This uses the byte units defined in ISO/IEC 80000-13:2008, both the binary prefixes and the original SI units.
+
+Supported units and abbreviations are as follows and are case-insensitive:
+
+### Metric/Decimal Prefixes
+
+|      Value       | Abbr |   Name    |
+| ---------------- | ---- | --------- |
+| 1                | B    | byte      |
```

Comment:
@dougwilson: ^^^^ Waiting for an answer here
Also, `1000^0` actually seems weird, just `1` would make more sense at first glance for newcomers.